# std_msgs/Header Header
rectbot_cv/PoseObject[] objects